### DISCUSSION TITLE
Do we have a plan to upgrade ISO 27001:2013 to ISO 27001:2022 certification?

### DISCUSSION THREAD

**Discussion by hqibec** [community/community#88511] [Enterprise]
### Select Topic Area

Question

### Body

As an enterprise user, our security team wants to know is there have an upgrade plan for ISO 27001 certification for Dotcom and GitHub Enterprise Cloud from 2013 to 2022?

Thank you!

**Comment by Shahnawazk7944** [comment]
I believe it will assist you
GitHub's ISO 27001 compliance information
https://github.blog/2022-05-16-github-achieves-iso-iec-270012013-certification/

Guide to upgrading to ISO 27001:2022
https://hyperproof.io/product/iso-27001/
